runtime/manual/tools/test.md

```diff
@@ -0,0 +1,326 @@
+
+- `--allow-net[=<IP_OR_HOSTNAME>...]` 
+    Allow [network access](https://deno.land/manual/basics/permissions). Optionally specify allowed IP addresses and host names, with ports as necessary.
+    Examples: `--allow-net`, `--allow-net="localhost:8080,deno.land"`
+
+- `--deny-net[=<IP_OR_HOSTNAME>...]`
+    Deny [network access](https://deno.land/manual/basics/permissions). Optionally specify denied IP addresses and host names, with ports as necessary.
+    Examples: `--deny-net`, `--deny-net="localhost:8080,deno.land"`
+
+- `--unsafely-ignore-certificate-errors[=<HOSTNAMES>...]`
+    DANGER: Disables verification of TLS certificates
+
+- `--allow-env[=<VARIABLE_NAME>...]`
+    Allow [access to system environment information](https://deno.land/manual/basics/permissions). Optionally specify accessible environment variables.
+    Examples: `--allow-env`, `--allow-env="PORT,HOME,PATH"`
+
+- `--deny-env[=<VARIABLE_NAME>...]`
+    Deny [access to system environment information](https://deno.land/manual/basics/permissions). Optionally specify accessible environment variables.
+    Examples: `--deny-env`, `--deny-env="PORT,HOME,PATH"`
+
+- `--allow-sys[=<API_NAME>...]`
+    Allow [access to OS information](https://deno.land/manual/basics/permissions). Optionally allow
+    specific APIs by function name.
+    Examples: `--allow-sys`, `--allow-sys="systemMemoryInfo,osRelease"`
+
+- `--deny-sys[=<API_NAME>...]`
+    Deny [access to OS information](https://deno.land/manual/basics/permissions). Optionally deny
+    specific APIs by function name.
+    Examples: `--deny-sys`, `--deny-sys="systemMemoryInfo,osRelease"`
+
+- `--allow-run[=<PROGRAM_NAME>...]`
+    Allow [running subprocesses](https://deno.land/manual/basics/permissions). Optionally
+    specify allowed runnable program names.
+    Examples: `--allow-run`, `--allow-run="whoami,ps"`
+
+- `--deny-run[=<PROGRAM_NAME>...]`
+    Deny [running subprocesses](https://deno.land/manual/basics/permissions). Optionally specify denied runnable program names.
+    Examples: `--deny-run`, `--deny-run="whoami,ps"`
+
+- `--allow-ffi[=<PATH>...]`
+    (Unstable) Allow loading dynamic libraries. Optionally specify [allowed directories or files](https://deno.land/manual/basics/permissions).
+    Examples: `--allow-ffi`, `--allow-ffi="./libfoo.so"`
+
+- `--deny-ffi[=<PATH>...]`
+    (Unstable) Deny [loading dynamic libraries](https://deno.land/manual/basics/permissions). Optionally specify denied directories or files.
+    Examples: `--deny-ffi`, `--deny-ffi="./libfoo.so"`
+
+- `--allow-hrtime`
+    Allow [high-resolution time measurement](https://deno.land/manual/basics/permissions). Note: this can enable timing attacks and fingerprinting.
+
+- `--deny-hrtime`
+    Deny [high-resolution time measurement](https://deno.land/manual/basics/permissions). Note: this can prevent
+  timing attacks and fingerprinting.
+
+- `-A, --allow-all`
+    Allow [all permissions](https://deno.land/manual/basics/permissions).
```

Comment:
I think all these permission flags should be in their own section.

Reply:
I was thinking the same, but couldn't quite decide where to put them. I'll update now.

Reply:
I've moved them into their own accordion for now.